compiler/next/include/chpl/queries/Context-detail.h

```diff
@@ -275,6 +289,14 @@ class QueryMapBase {
    const char* queryName;
    bool isInputQuery;
 
+  struct QueryStats {
+    QueryTimingStat query;
+    // `system` here refers to what the query system is doing
+    // `self` is confusing b/c a profiler uses `self` to mean, time in me - time in children
+    QueryTimingStat systemGetMap;
+    QueryTimingStat systemGetResult;
```

Comment:
I see by looking at the code later that these are timing getMap and getResult functions. Could you put some comments here to that effect? And maybe say something like - we can add timing for other specific functions here if we are curious about their performance.

Reply:
Updated with more comments